mw-webapp/src/logic/wayPage/reportsTable/WayColumns.tsx

```diff
@@ -50,10 +50,23 @@ interface RenderModalContentParams {
  * TODO: use modal instead of confirm task #305
  */
 export const renderModalContent = (params: RenderModalContentParams) => {
-  const isUserWantToDeleteJobDone = confirm(`Are you sure that you want to delete jobDone "${params.description}"?`);
+  const isUserWantToDeleteJobDone = confirm(params.description);
   !!isUserWantToDeleteJobDone && params.onOk();
 };
 
+/**
+ * Handle onClick
+ */
+export const handleOnClick = (description: string, handleOnOk: () => void) => {
```

Comment:
this functions not hadled onClick. It started process of removing some entity
useless wrapper

Reply:
Deleted